src/internal/ppsutil/util.go

```diff
@@ -117,8 +117,9 @@ func GetLimitsResourceList(limits *pps.ResourceSpec) (*v1.ResourceList, error) {
 func GetPipelineDetails(pachClient *client.APIClient, pipelineInfo *pps.PipelineInfo) error {
 	// ensure we are authorized to read the pipeline's spec commit, but don't propagate that back out
 	pachClient = pachClient.WithCtx(pachClient.Ctx())
-	pachClient.SetAuthToken(pipelineInfo.AuthToken)
-
+	if pipelineInfo.AuthToken != "" {
+		pachClient.SetAuthToken(pipelineInfo.AuthToken)
```

Comment:
I don't think this code is necessary for metrics to work anymore?

Reply:
You're correct, I think this is necessary for the pipeline controller. Happy to just set it outside of this function in that case, though.